packages/ds-ext/src/app/VsCodeClient.tsx

```diff
@@ -36,11 +39,20 @@ export class VsCodeClient implements WorkspaceApiClient {
     this.sendMessage(message);
   }
 
-  async getTree() {}
-  async createTree() {}
-  async updateTree() {}
-  async destroyTree() {}
-  async moveTree() {}
+  async getTree() { }
+  async createTree() { }
+
+  async updateTree(diagram: Diagram) {
```

Comment:
Since we deprecated the tree concept, can we rename this method to updateDiagram or similar?
If this is interfering with old Interfaces, lets update them (in this PR or follow up)